charts/ggscout/templates/clusterrole.yaml

```diff
@@ -13,4 +13,14 @@ rules:
   - apiGroups: [""]
     resources: ["secrets"]
     verbs: ["get", "list"]
+  - apiGroups: [ "" ]
+    resources: ["configmaps"]
+    verbs: ["get", "list"]
+  - apiGroups: ["apps"]
+    resources: ["deployments"]
+    verbs: ["get", "list"]
+  - apiGroups: ["kubernetes-client.io"]
```

Comment:
why did you change this? from `kubectl api-resources` we get this 

<img width="1234" alt="Screenshot 2025-04-04 at 10 39 04" src="https://github.com/user-attachments/assets/e61f9c28-b679-42f4-bcd5-1da70b33306e" />

Reply:
There are also some `external-secrets.io` as apiGroup, I think both are OK